### PR TITLE
refactor: update stack with component-rollup-config

### PR DESCRIPTION
**Pull request by cbourgois**
### Description of the Change

replace module name from `ovh-angular-responsive-tabs` to `ngOvhResponsiveTabs`
BREAKING CHANGE: module is now named `ngOvhResponsiveTabs`

5cf0ec4 — refactor: update stack with component-rollup-config

### TODO

* [x] rename repository to `ng-ovh-responsive-tabs`

/cc @jleveugle @frenautvh @antleblanc

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,6 +1,3 @@
 node_modules
-.tmp
-.idea
-npm-debug.log
-bower_components
-.work
+dist
+tools
```
Comment (antleblanc) [inline .gitignore]: -`tools` (not needed)

File: .lintstagedrc.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+  "*.js": [
```
Comment (antleblanc) [inline .lintstagedrc.json:5]: missing `*.html` (using `htmlhint`)

File: package.json [hunk 3]
```diff
@@ -1,40 +1,56 @@
...
+  "name": "@ovh-ux/ng-ovh-responsive-tabs",
   "version": "4.0.0",
   "description": "Wrapper for Angular UI Bootstrap directive 'Tabs'",
-  "main": "./dist/ovh-angular-responsive-tabs.min.js",
-  "repository": "ovh-ux/ovh-angular-responsive-tabs",
-  "scripts": {
-    "test": "grunt test"
-  },
+  "keywords": [
+    "OVH",
```
Comment (antleblanc) [inline package.json]: prefer lowercase.

File: package.json [hunk 4]
```diff
@@ -1,40 +1,56 @@
...
+    "dist"
+  ],
+  "main": "./dist/cjs/index.js",
+  "module": "./dist/esm/index.js",
+  "browser": "./dist/umd/ng-ovh-responsive-tabs.js",
+  "scripts": {
+    "build": "rollup -c",
+    "dev": "rollup -c --watch",
+    "lint": "run-p lint:js lint:md",
+    "lint:js": "eslint --quiet --fix --format=pretty src",
```
Comment (antleblanc) [inline package.json:25]: missing `lint:html`

File: package.json [hunk 5]
```diff
@@ -1,40 +1,56 @@
...
-    "jasmine-core": "^2.6.4",
-    "karma": "~0.13.22",
-    "karma-jasmine": "~0.3.0",
-    "karma-phantomjs-launcher": "~1.0.4",
-    "karma-ng-html2js-preprocessor": "~0.1.0",
-    "phantomjs-prebuilt": "^2.1.14",
-    "matchdep": "^1.0.0"
+    "@commitlint/cli": "^7.1.2",
+    "@commitlint/config-angular": "^7.1.2",
+    "@ovh-ux/component-rollup-config": "5.0.0-beta.5",
```
Comment (antleblanc) [inline package.json]:
```suggestion
    "@ovh-ux/component-rollup-config": "5.0.0-beta.6",
```

File: src/more/index.js [hunk 6]
```diff
@@ -0,0 +1,11 @@
+import angular from 'angular';
+
+import directive from './directive';
+
+const moduleName = 'ngOvhResponsiveTabsMore';
+
+angular
+  .module(moduleName, [])
+  .directive('responsiveTabMore', directive);
```
Comment (antleblanc) [inline src/more/index.js]: What's the purpose of creating a new module that contain the `responsiveTabMore` directive?

Why not adding that directive to the `ngOvhResponsiveTabs` module?

File: src/tab/index.js [hunk 7]
```diff
@@ -0,0 +1,11 @@
+import angular from 'angular';
+
+import directive from './directive';
+
+const moduleName = 'ngOvhResponsiveTabsTab';
+
+angular
+  .module(moduleName, [])
+  .directive('responsiveTab', directive);
```
Comment (antleblanc) [inline src/tab/index.js]: What's the purpose of creating a new module that contain the `responsiveTab` directive?

Why not adding that directive to the `ngOvhResponsiveTabs` module?

**Review by antleblanc — CHANGES_REQUESTED** [review]
:warning: `yarn-error.log` file must be ignored.

File: package.json (hunk 5 above)
Comment (cbourgois) [reply package.json]: Bumped to 

```suggestion
    "@ovh-ux/component-rollup-config": "5.0.0-beta.8",
```

File: src/more/index.js (hunk 6 above)
Comment (cbourgois) [reply src/more/index.js]: Sure, modules `tab` & `more` deleted.

File: src/tab/index.js (hunk 7 above)
Comment (cbourgois) [reply src/tab/index.js]: Sure, modules `tab` & `more` deleted.

**Review by antleblanc — APPROVED** [review]

**Pull request merged**